examples/04_manipulating_images/plot_roi_extraction.py

```diff
@@ -307,7 +308,7 @@
 # Preparing for data extraction: setting number of conditions, size, etc from
 # haxby dataset
 condition_names = list(set(haxby_labels))
-n_cond_img = fmri_data[..., haxby_labels == b'house'].shape[-1]
+n_cond_img = fmri_data[..., haxby_labels == 'house'].shape[-1]
```

Comment:
Would "(haxby_labels == 'house').sum()" give the same result?

Reply:
This gives a different result, actually, so I'm keeping with the same code for now!